botorch/optim/numpy_converter.py

```diff
@@ -42,6 +45,21 @@ class TorchAttr(NamedTuple):
     device: torch.device
 
 
+def _handle_numerical_errors(
+    error: RuntimeError, x: np.ndarray
+) -> Tuple[float, np.ndarray]:
+    if isinstance(error, NotPSDError):
+        raise error
+    error_message = error.args[0] if len(error.args) > 0 else ""
+    if (
+        isinstance(error, NanError)
+        or "singular" in error_message  # old pytorch message
+        or "input is not positive-definite" in error_message  # since pytorch #63864
```

Comment:
#63864 has long made it into the releases and is part of 1.11 our min version requirement. 

```suggestion
        or "input is not positive-definite" in error_message
```